emote.py

```diff
@@ -46,9 +57,12 @@ def emoteadd(self, emote):
 
 		return 'Emote "' + args[0] + '" added :)'
 
+	def emotelist(self):
+		return 'Emotes: ' + ', '.join(list(self.emotes.keys()))
+
 	def emotedel(self, emote):
 		if emote == 'lenny':
-			return 'One cannot simply delete Lenny ( ͡° ͜ʖ ͡°)'
+			return 'One cannot simply delete Lenny ' + self.emotes['lenny'][0]
```

Comment:
Actually, you do this on line 82. Up to you if it matters to keep it the same or not